### DISCUSSION TITLE
mc.exe: <ERROR> Unable to stat source `ABC/Bucket1`.

### DISCUSSION THREAD

**Discussion by louiskoh** [minio/minio#19148] [Q&A]
Install/Create the MinIO server with "UserA" logon to the server. "UserA" is able to perform backup by running "mc mirror ABC/Bucket1 'C:\MinIO Backup'" command.

Logon to the server using "UserB", try to perform backup by running the same command as above but it failed and show below error:
mc.exe: <ERROR> Unable to stat source `ABC/Bucket1`. Requested path `C:\Program Files\PIA\bin\BackupRestoreTool\ABC\Bucket1` not found.

Is it true that MinIO somehow remember/recognize only that particular user which used to create/install the MinIO server?
Is there anyway to allow other user (other than"UserA") to perform "mc mirror ABC/Bucket1 'C:\MinIO Backup'" command successfully?

Thank.

**Comment by klauspost** [comment]
Aliases are stored per user. For example for user klaus it is at `c:\Users\klaus\mc\config.json`

You can use `--config-dir` to specify a custom dir.
